README.md

```diff
@@ -178,6 +198,63 @@ make -k TOOLCHAIN=clang-18  CONFIG=Tsan  VERBOSE=1
 
 The makefile will use your system compiler, `c++`, if no toolchain name is provided, otherwise it will use the toolchain in the etc/ directory to perform the build. The Ninja multi config generator is used, with configurations for `RelWithDebugInfo`, `Debug`, `Tsan`, and `Asan` configured by default.
 
+#### Step by Step Build: Build and Run Tests
+
+CI current build and test flows:
+
+```shell
+# Configure build: default build production code + tests (BUILD_TESTING=ON by default).
+$ rm -rf .build/
+$ mkdir -p .build
```

Comment:
I don't think this command is necessary. cmake will create the directory IIUC.

Reply:
Indeed, simplified to:

CI current build and test flows:

```shell
# Configure build: default build production code + tests (BUILD_TESTING=ON by default).
$ cmake -G "Ninja Multi-Config" -DCMAKE_CONFIGURATION_TYPES="RelWithDebInfo;Asan" -DCMAKE_TOOLCHAIN_FILE=etc/clang-19-toolchain.cmake -B .build -S .
-- The CXX compiler identification is Clang 19.0.0
...
-- Build files have been written to: /path/to/Optional26/.build

# Build.
$ cmake --build .build --config Asan --target all -- -k 0
...
[30/30] Linking CXX executable ...

# Run tests.
$ ctest --build-config Asan --output-on-failure --test-dir .build
Internal ctest changing into directory: /path/to/Optional26/.build
Test project /path/to/Optional26/.build
...
100% tests passed, 0 tests failed out of 82

Total Test time (real) =   0.67 sec
```

Thanks!